docs/api/algorand/create-delegated-payment.mdx

```diff
@@ -0,0 +1,105 @@
+
+Returned when the parameters are invalid
+
+| Name | Content | 
+| ---- | ------- | 
+| Status code | 400 |
+| errorId | `INVALID_PARAMS` |
+| message | `Validation errors in CreateDelegatedPayment request.` | 
+| errors | Array of strings containing additional informations |
+
+
+### Sender not opted in 
+
+Returned when the sender has not yet opted into the token that is being sent.
+
+| Name | Content | 
+| ---- | ------- | 
+| Status code | 400 |
+| errorId | `SENDER_NOT_OPTED_IN` |
+| message | `Sender must opt in to the asset before sending funds.` | 
+
+### Sender not enough funds
+
+Returned when the sender does not have enough funds to pay both the transfer of the funds and the fee.
+
+| Name | Content |
+| ---- | ------- |
+| Status code | 400 |
+| errorId | `SENDER_NOT_ENOUGH_FUNDS` |
+| message | `Sender does not have enough funds to send the requested amount.` |
+
+### Receiver not opted in
+
+Returned when a user is sending funds to a non-[DelegatedWallet](#CreateDelegatedPayment) account that has not yet opted into the token that is being sent.
+
+| Name | Content |
+| ---- | ------- |
+| Status code | 400 |
+| errorId | `RECEIVER_NOT_OPTED_IN` |
+| message | `Receiver must opt in to the asset before receiving funds.` |
+
+
+
+## Example Usage
+
+<Tabs>
+<TabItem value="js" label="JavaScript">
+
+```js
+const response = await fetch('https://api.decash.com/CreateDelegatedPayment', {
+  method: 'POST',
+  body: JSON.stringify({
+    assetId: 'usd',
+    sender: {
+      address: 'DUIUUAOHRAUHVE5GXDE6OURUFWVXXFNSCOK3IMGWCZWSAASSM342O4YMAY',
+      amount: 14.50,
+    },
+    receiver: 'CNIOPGMZ3OSZLPEO7POGMVUEQVDV2RT3G4MSKGHRBX2SOPOEIOGBIFIMVQ',
+  })
+});
+const data = await response.json();
+
+```
+
+</TabItem>
+</Tabs>
+
+## Example Success Response
+
+```json
+{
+  "noSign": ["iqNhbXTOAAMNQKNmZWXND6CiZnbOARq8HqNnZW6sdGVzdG5ldC12MS4womdoxCBIY7UYpLPITsgQ8i1PEIHLD3HwWaesIN7GL39w5Qk6IqNncnDEINiYMg4zFOLDjNerIPaxDIAcHhwOCGtftTZmF1PUxRzSomx2zgEawAajcmN2xCCJZZGRCVyrMfyFvsxeVLYvYG5hdipXoAQLRYjmhvjjJqNzbmTEIOFCRKufkeV8Sz20fmzi1S0BI6tryNDZqb/HW3vtf2fkpHR5cGWjcGF5","iaRhcmN2xCCJZZGRCVyrMfyFvsxeVLYvYG5hdipXoAQLRYjmhvjjJqJmds4BGrweo2dlbqx0ZXN0bmV0LXYxLjCiZ2jEIEhjtRiks8hOyBDyLU8QgcsPcfBZp6wg3sYvf3DlCToio2dycMQg2JgyDjMU4sOM16sg9rEMgBweHA4Ia1+1NmYXU9TFHNKibHbOARrABqNzbmTEIIllkZEJXKsx/IW+zF5Uti9gbmF2KlegBAtFiOaG+OMmpHR5cGWlYXhmZXKkeGFpZM4DGUVQ"],
+  "toSign": ["iqRhYW10AaRhcmN2xCDhQkSrn5HlfEs9tH5s4tUtASOra8jQ2am/x1t77X9n5KJmds4BGrweo2dlbqx0ZXN0bmV0LXYxLjCiZ2jEIEhjtRiks8hOyBDyLU8QgcsPcfBZp6wg3sYvf3DlCToio2dycMQg2JgyDjMU4sOM16sg9rEMgBweHA4Ia1+1NmYXU9TFHNKibHbOARrABqNzbmTEIAZekVF1hNOHnM9lYgORVqhWh57VRV/Cwtzz71pasQgJpHR5cGWlYXhmZXKkeGFpZM4DGUVQ","iqRhYW10CqRhcmN2xCCJZZGRCVyrMfyFvsxeVLYvYG5hdipXoAQLRYjmhvjjJqJmds4BGrweo2dlbqx0ZXN0bmV0LXYxLjCiZ2jEIEhjtRiks8hOyBDyLU8QgcsPcfBZp6wg3sYvf3DlCToio2dycMQg2JgyDjMU4sOM16sg9rEMgBweHA4Ia1+1NmYXU9TFHNKibHbOARrABqNzbmTEIAZekVF1hNOHnM9lYgORVqhWh57VRV/Cwtzz71pasQgJpHR5cGWlYXhmZXKkeGFpZM4DGUVQ"],
```

Comment:
For my knowledge: what happens if the signed transaction is not submitted to the network, but later (say, 3 years later) some admin finds it and decides to submit?

- Will it be submitted?
- Does this signature has some expiration date?
- Can you re-submit the transaction once again if it was submitted?

Reply:
Transactions have a firstValid and lastValid field, so the transaction will expire after a bit. Also it won't be possible to submit it twice, as it has a particular transaction ID (which must be unique).

Reply:
Thanks!

> firstValid and lastValid field

Is it provided by Algorand (core)?

Reply:
Yep
https://developer.algorand.org/docs/get-details/transactions/transactions/